.github/actions/setup-java-build/action.yml

```diff
@@ -19,8 +23,12 @@ runs:
         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-maven-
+    - name: "Install ${{ inputs.maven-settings }}"
+      shell: bash
+      run: "cp ${{ inputs.maven-settings }} $HOME/.m2/settings.xml || :"
```

Comment:
add a `mkdir $HOME/.m2/` otherwise will fail when there is not a cache to be restored

I don't see `:` used since long time, prefer the built-in `true` for more clarity

Reply:
Done, thanks!